docs/_sass/custom/custom.scss

```diff
@@ -183,7 +216,7 @@ body {
 
 	pre code {
 		background-color: $darkblack;
-    	border: 0px solid $lightgreen;
+    		border: 0px solid $lightgreen;
```

Comment:
indent

Reply:
hmm, can you do that at your end? just remove extra space etc

Reply:
no, you will need to push the changes to this pull request.
also, this like is just an example there are a couple of places here where you used tab vs spaces for example. Please make sure you have all the styles aligned.